exp/production.md

```diff
@@ -0,0 +1,84 @@
+* Where is your AMS Application Registry (AAR) deployed? How will the AMS in other clusters connect to AAR? Define your VPC peering requirements based on the number of clusters you have and where AAR is deployed.
+
+## Storage requirements
+Based on your safety protocols, think about where you would want to host your storage.
+
+## Scalability, High availability, and redundancy
+
+With Anbox Cloud, you can scale your deployment to as many clusters and subclusters as you require. It is a good idea to define how many Anbox clusters you want to deploy and how many subclusters per cluster. 
+
+You should also define the number of instances that you require for each model/service. For example, think about the number of Juju controller instances that you need.
+
+Anbox Cloud comes with support for High Availability (HA) for both Core and the Streaming Stack.You can define HA by adding new Juju units. See [How to enable High availability?](https://anbox-cloud.io/docs/howto/install/high-availability) to plan your HA requirements.
+
+## Security
+
+### Control Plane TLS Termination
+
+The components of Anbox Cloud need to be able to communicate securely over the network. This is accomplished using TLS and public-key encryption with a chain of trust up to a shared root Certificate Authority (CA). However, when the cluster is being brought up or a new unit is being added, the chain of trust and certificates required must be bootstrapped into the machines.
+
+All communication between Juju units and the Juju controller happens over TLS-encrypted websockets. The certificate for the TLS connection to the controller is added as explicitly trusted to each machine as part of the bootstrap process using a combination of cloud-init and SSH.
+
+With this secure channel, Juju charms can communicate with each other using relation data. The data published by one unit is sent to the controller, which then makes it available for all other units on the same relation. The data for each relation is scoped by ID and is only visible to units participating in the specific relation on which it is set.
+
+### Patching
+
+You should keep your Anbox deployment updated to the latest available stable version. You should also update the other applications which make up Anbox. Keeping up to date ensures you have the latest fixes and security patches for smooth operation of your cluster.
+For details about Anbox release roadmap can be found here:
+https://anbox-cloud.io/docs/roadmap
+
+## Data backup
+
+Consider how frequently you want your data backed up and where you want to host your backed up data.
+
+## Load balancing 
+
+Load balancing is done using the Elastic Load Balancing from AWS. Assess how many load balancers you require and whether you want to place them on public or private subnets. You should also think about whether you want to associate these load balancers with Elastic IPs.
+
+## Monitoring and metrics
```

Comment:
I am not so sure about this part. Below is the old stack which we should not point to much longer. https://charmhub.io/topics/canonical-observability-stack is what we will use in the future. 

Maybe we stay rather generic here? Wdyt?